asserts/asserts.go

```diff
@@ -872,6 +873,31 @@ func Assemble(headers map[string]interface{}, body, content, signature []byte) (
 	return assemble(headers, body, content, signature)
 }
 
+func checkAuthority(assertType *AssertionType, headers map[string]interface{}) (hasSignatoryID bool, err error) {
```

Comment:
Parameter is unused
```suggestion
func checkAuthority(headers map[string]interface{}) (hasSignatoryID bool, err error) {
```

Reply:
I think I have it for symmetry with CheckNoAuthority, I made it clearer (with _) that is unused though